presto-docs/src/main/sphinx/release/release-0.251.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.251
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Improve dictionary writer performance.
+* Add BigQuery Connector Plugin.
+* Added support for memozing dead nodes. This can be enabled by setting the ``internal-communication.memoize-dead-nodes-enabled`` configuration property.
+* Enforce NOT NULL constraint by checking values for NOT NULL columns.
```

Comment:
We should use one of the words in this section to start a comment - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes

```
Add support to enforce ``NOT NULL`` constraint by checking values for ``NOT NULL`` columns
```